test/lib/qb_backend/posts/bookmark_test.exs

```diff
@@ -0,0 +1,50 @@
+defmodule QbBackend.Posts.BookmarkTest do
+  @moduledoc """
+  this module comprises for tests for the bookmark model
+  """
+
+  use QbBackend.DataCase
+
+  alias QbBackend.{
+    Posts.Bookmark,
+    Posts.Manual,
+    Repo
+
+    }
+    @valid_bookmark %{}
+  describe "Bookmark" do
+    test "Enable a user to create a bookmark" do
```

Comment:
@WanjikuMac  this test actually just `test Drives` the bookmark changeset

Reply:
@zacck This checks if a bookmark is actually created.